src/generate.jl

```diff
@@ -15,14 +15,12 @@ function generate(
     model::Transformer{T},
     tokenizer::Tokenizer,
     sampler::Sampler{T},
-    prompt::String,
+    prompt::String;
```

Comment:
Why did you choose to go with a different separator for this parameter? Not complaining about functionality, just curious about the design decision.

Reply:
https://docs.julialang.org/en/v1/manual/functions/#Keyword-Arguments